src/main/scala/org/broadinstitute/hail/variant/VariantSampleMatrix.scala

```diff
@@ -294,6 +294,10 @@ object VSMSubgen {
     vGen = Variant.gen,
     tGen = Genotype.genExtreme)
 
+  val plinkSafeSingleAlt = random.copy(
```

Comment:
I think this should be renamed to `plinkSafeBiallelic`, and should gen.map the vds to have `wasSplit = true` at the end